config_standard_storage.yaml

```diff
@@ -18,15 +18,20 @@ proxy:
 singleuser:
   # very small limits for testing on small instances
   memory:
-    guarantee: 500M
-    limit: 1G
+    guarantee: 10G
+    limit: 10G
   cpu:
-    guarantee: 0.3
-    limit: 1
+    guarantee: 4
+    limit: 10
   storage:
     type: dynamic
-    capacity: 1Gi
+    capacity: 5Gi
     dynamic:
       storageClass: standard
+  image:
+    name: supercdms/cdms-jupyterlab
+    tag: 1.8b
```

Comment:
in you files you had `0.7b` which doesn't exist at: https://hub.docker.com/r/supercdms/cdms-jupyterlab/tags
I am trying with `1.8b`

Reply:
@pibion